templates/app/models/sequelize-model.hbs

```diff
@@ -20,20 +20,41 @@ module.exports = (sequelize, DataTypes) => {
     schema: process.env.DATABASE_SCHEMA,{{/if}}
   });
 {{#if noId}}  {{modelName}}.removeAttribute('id');{{/if}}
-  // This section contains the relationships for this model. See: https://docs.forestadmin.com/documentation/v/v5/reference-guide/relationships#adding-relationships.
```

Comment:
It seems this line should be present.

Reply:
This is a diff between today lumber generation (left) and this branch lumber generation (right)
<img width="980" alt="Screenshot 2019-12-11 at 14 04 09" src="https://user-images.githubusercontent.com/3200482/70623803-26476b00-1c1f-11ea-9cef-cc2f326f70c9.png">

`field:''` is present on all lines and is not desired
the `as` property is not present and is desired
The comment is is not present and is desired.

Reply:
For your information, another shot with old lumber at left and this branch lumber at right

FOREST_URL="https://api.development.forestadmin.com" NODE_TLS_REJECT_UNAUTHORIZED=0 npm start